db/seeds.rb

```diff
@@ -145,74 +145,205 @@
+    question_type: "Radio button"
   },
   {
     text: "In which of the following contexts have you done significant amounts of professional conservation work?",
+    options: ["Very heavily modified ecosystems (e.g. urban areas)",
+              "Largely modified ecosystems (e.g. intensive farmland, intensive fisheries)",
+              "Somewhat modified ecosystems (e.g. low intensity farmland, low intensity fisheries)",
+              "Largely unmodified ecosystems (e.g. sustainable use area)",
+              "Unmodified or very little modified ecosystem (e.g. wilderness areas)",
+              "Other",
+              "Not applicable"],
+    question_type: "Checkbox"
   },
   {
     text: "Do you have any experiences of working as a researcher or as a practitioner on market based schemes in conservation (e.g. payments for ecosystem services, taxes and subsidies, certification)?",
+    options: ["Yes, as a researcher",
+              "Yes, as a practitioner",
+              "Yes, both as a researcher and as a practitioner",
+              "No"],
+    question_type: "Radio button"
   },
   {
     text: "Please choose up to four of the following items that you believe have been most important in shaping your conservation values",
+    options: ["Values of family and friends during childhood",
+              "Values of family and friends during adulthood",
+              "Outdoor play/recreation during childhood",
+              "Outdoor play/recreation during adulthood",
+              "Indoor experiences (e.g. reading books, watching documentaries, playing video games) during childhood",
+              "Indoor experiences (e.g. reading books, watching documentaries, playing video games) during adulthood",
+              "Formal education experience (e.g. field trip, classroom session) during childhood",
+              "Formal education experience (e.g. field trip, classroom session) during adulthood",
+              "Professional experience during adulthood",
+              "None of the above",
+              "Not applicable"],
+     question_type: "Checkbox"
   },
   {
     text: "Please feel free to tell us about anything else that you think was important in shaping your values",
+    question_type: "Free Text"
   },
   {
     text: "Please provide us with your email address so that we can send you summarised results and keep in contact with you about future work based on this survey.\\r\\n\\r\\nWe will not use your address for any other reason.",
+    question_type: "Free Text"
   },
   {
     text: "Have you taken this survey before?",
+    options: ["Yes", "No"],
+    question_type: "Radio button"
   },
   {
     text: "Are you taking part in the WWF UK staff survey?",
+    options: ["Yes", "No"],
+    question_type: "Radio button"
   },
   {
     text: "If you are answering this as part of the WWF UK survey please choose your programme",
+    options: ["Yes", "No"],
+    question_type: "Radio button"
   },
```

Comment:
This answer does not make sense as a "Yes" or "No" but I thought I had copied it exactly from Future of Conservation.

Reply:
Looks we have a slightly different question on production: “Are you part of the WWF Senior Science Team survey?”

Reply:
Yeah let's remove that question